libr/core/cconfig.c

```diff
@@ -2405,6 +2405,8 @@ R_API int r_core_config_init(RCore *core) {
 	SETPREF ("asm.emu.stack", "false", "Create a temporary fake stack when emulating in disasm (asm.emu)");
 	SETCB ("asm.emu.str", "false", &cb_emustr, "Show only strings if any in the asm.emu output");
 	SETPREF ("asm.emu.stroff", "false", "Always show offset when printing asm.emu strings");
+	SETPREF ("asm.emu.strinvert", "true", "If true, color-invert asm.emu strings and not directly "
```

Comment:
tldr, use shorter descriptions. and im afraid we dont want so many vars under asm.

maybe we should go for asm.emu.str.* if we have more asm.emu.str.* options..

Reply:
i think shortening asm.emu.strinvert to asm.emu.strinv is better. Those dots can be a bit of a pain to type in.

i've committed a shorter description in 689269bcc2a32af8a8dc916a8ad56cf9dcefcc40.